client/Importers/StatBlockImporter.ts

```diff
@@ -1,14 +1,20 @@
 import { StatBlock } from "../../common/StatBlock";
 import { AccountClient } from "../Account/AccountClient";
 import { Importer } from "./Importer";
+import titleCase = require("title-case");
```

Comment:
Lodash has a [_.startCase()](https://lodash.com/docs/4.17.10#startCase) method that I think covers this, removing the need for the dependency.